src/controllers/roleControllers.ts

```diff
@@ -0,0 +1,71 @@
+import logger from '../logs/config';
+import { Request, Response } from 'express';
+import Role from '../database/models/role';
```

Comment:
All models should come from 
`import Role from '../database/models';` or `import Role from '../database/models/index.ts';`

Reply:
since the models are not related to the configurations in `models/index.ts`, I am really failing to make sure that every mdoel comes from this file `index.ts`. 
Will I have to import models to this file and then export them? If so, what is the purpose of these redundancy? Please enlighten me! Thanks!

For clarification, this is how my `models/index.ts` looks like:
```
'use strict';

import { Sequelize, Dialect } from 'sequelize';

const env: string = process.env.NODE_ENV || 'development';

const config = require('../config/config.js');

const dbConfig = config[env];

const sequelize = new Sequelize(dbConfig.database!, dbConfig.username!, dbConfig.password!, {
  dialect: dbConfig.dialect! as Dialect,
  host: dbConfig.host || 'localhost',
  port: dbConfig.port,
});

export default sequelize;
````